source/Scripts/CompileRootConfiguration.ps1

```diff
@@ -21,6 +21,20 @@ Sampler.DscPipeline:
     Version: 0.3.259
   - PSDesiredStateConfiguration
 #>
+
+if (-not $BuildInfo.'Sampler.DscPipeline')
+{
+    Write-Error "There are no modules to import defined in the 'build.yml'. Expected the element 'Sampler.DscPipeline'"
```

Comment:
```suggestion
    Write-Error -Message "There are no modules to import defined in the 'build.yml'. Expected the element 'Sampler.DscPipeline'"
```

Reply:
Should we not use the key 'SamplerDscPipeline' to avoid the needs of quotes in the yaml? I don't know honestly.
DscPipeline could also be enough?

Reply:
There are quite a few dependencies with a dot in the name. I don't mind using quotes in these cases.